engine/graphics/src/dmsdk/graphics/graphics.h

```diff
@@ -439,6 +439,19 @@ namespace dmGraphics
         BLEND_FACTOR_ONE_MINUS_CONSTANT_ALPHA = 14,
     };
 
+    /*#
+     * Vertex step function. Dictates how the data for a vertex attribute should be read in a vertex shader.
+     * @enum
+     * @name VertexStepFunction
+     * @member VERTEX_STEP_FUNCTION_VERTEX
+     * @member VERTEX_STEP_FUNCTION_INSTANCE
+     */
+    enum VertexStepFunction
+    {
+        VERTEX_STEP_FUNCTION_VERTEX,
+        VERTEX_STEP_FUNCTION_INSTANCE,
+    };
```

Comment:
We already support instancing in the vulkan API because of extensions (and we need it soon for engine wide support anyway)